rustls/Cargo.toml

```diff
@@ -17,6 +17,7 @@ rustversion = { version = "1.0.6", optional = true }
 
 [dependencies]
 aws-lc-rs = { version = "1.6", optional = true, default-features = false, features = ["aws-lc-sys"] }
+brotli = { version = "6", optional = true, default-features = false, features = ["std"] }
```

Comment:
Seems reasonable to not worry about a no-std compatible impl of this at this stage :+1:

Reply:
I couldn't work out how to use this crate without std, basically :)